docs/blog/posts/move-to-litellm-proxy.md

```diff
@@ -0,0 +1,25 @@
+---
+authors:
+  - Amir
+category:
+  - Architecture
+
+date: 2024-04-05
+---
+# Model calls made flexible
```

Comment:
Suggestion: "Adding a model proxy server"? 
But it's your decision in the end.

Reply:
agreed